src/ctree-segments/ctree-segment-image.html

```diff
@@ -163,10 +166,11 @@
 
       _editSaveClicked() {
         super._addSegmentVariation(this.$.image.src);
+        super._closure();
```

Comment:
I think you can use `this._closure()` instead of `super._closure()`, can't you?  Usually `super` is only used when you're overriding a function, because you don't want to short circuit your ability to override the functionality.